src/main/java/de/tum/in/www1/artemis/web/rest/tutorialgroups/TutorialGroupFreePeriodResource.java

```diff
@@ -244,4 +247,14 @@ private void trimStringFields(TutorialGroupFreePeriod tutorialGroupFreePeriod) {
             tutorialGroupFreePeriod.setReason(tutorialGroupFreePeriod.getReason().trim());
         }
     }
+
+    /**
+     * Used because we want to interpret the date in the time zone of the tutorial groups configuration
+     *
+     * @param startDate
+     * @param endDate
+     * @param reason
+     */
+    public record TutorialGroupFreePeriodDTO(@NotNull LocalDateTime startDate, @NotNull LocalDateTime endDate, @JsonInclude(JsonInclude.Include.NON_NULL) String reason) {
```

Comment:
I would still add it into a different file. We typically use `@JsonInclude(JsonInclude.Include.NON_EMPTY)` above the class/record